docs/dev/style-guide.rst

```diff
@@ -200,9 +202,17 @@ Explanation
 How-to guides
 ~~~~~~~~~~~~~
 
-* Title should begin with **“How to ...”**,
-  but navigation titles should not contain the “How to” part.
+* Title should begin with **“How to ...”**.
+  If the how-to guide is specific for Sphinx, make sure to note it in the title.
```

Comment:
```suggestion
  If the how-to guide is specific for a tool, make sure to note it in the title.
```

Reply:
Nice one!

I have a weird love when the style guide doesn't follow the style guide, it so perfectly fits that old saying of "do as I say not as I do" :)